apps/remote_control/lib/lexical/remote_control/api.ex

```diff
@@ -60,4 +60,10 @@ defmodule Lexical.RemoteControl.Api do
       position
     ])
   end
+
+  def docs(%Project{} = project, module) when is_atom(module) do
+    with {:module, _} <- RemoteControl.call(project, Code, :ensure_compiled, [module]) do
```

Comment:
I'm not sure that this `Code.ensure_compiled/1` call is necessary. I was seeing some strange behavior in the hover tests, where the first call to `Code.fetch_docs/1` was returning an error, but a second call was returning the docs -- in my experience, stuff like that usually happens when something hasn't been fully loaded/etc. So this might really just be a "fix" for a problem in the test setup for the hover provider.

Reply:
I believe this is necessary because we cannot guarantee that the module is compiled when the user calls the hover feature, especially when we open the editor and the project compilation time is long.

Reply:
these two functions feel a bit like business logic that should go inside `RemoteControl` somewhere.